diff --git a/addons/stock/models/stock_move.py b/addons/stock/models/stock_move.py
index e8095abb96e6..d91af1903354 100644
--- a/addons/stock/models/stock_move.py
+++ b/addons/stock/models/stock_move.py
@@ -407,10 +407,6 @@ class StockMove(models.Model):
             precision_digits = self.env['decimal.precision'].precision_get('Product Unit of Measure')
             qty = float_round(move.quantity_done, precision_digits=precision_digits, rounding_method='HALF-UP')
             if float_compare(uom_qty, qty, precision_digits=precision_digits) != 0:
-                err.append(_("""
-The quantity done for the product %s doesn't respect the rounding precision defined on the unit of measure %s.
-Please change the quantity done or the rounding precision of your unit of measure.""",
-                             move.product_id.display_name, move.product_uom.display_name))
                 continue
             delta_qty = move.quantity_done - move._quantity_done_sml()
             if float_compare(delta_qty, 0, precision_rounding=move.product_uom.rounding) > 0:
diff --git a/addons/stock/models/stock_move_line.py b/addons/stock/models/stock_move_line.py
index 16a1e02f4e0f..6723fde62223 100644
--- a/addons/stock/models/stock_move_line.py
+++ b/addons/stock/models/stock_move_line.py
@@ -540,9 +540,7 @@ class StockMoveLine(models.Model):
             precision_digits = self.env['decimal.precision'].precision_get('Product Unit of Measure')
             qty_done = float_round(ml.qty_done, precision_digits=precision_digits, rounding_method='HALF-UP')
             if float_compare(uom_qty, qty_done, precision_digits=precision_digits) != 0:
-                raise UserError(_('The quantity done for the product "%s" doesn\'t respect the rounding precision '
-                                  'defined on the unit of measure "%s". Please change the quantity done or the '
-                                  'rounding precision of your unit of measure.') % (ml.product_id.display_name, ml.product_uom_id.name))
+                continue
 
             qty_done_float_compared = float_compare(ml.qty_done, 0, precision_rounding=ml.product_uom_id.rounding)
             if qty_done_float_compared > 0:
